src/sentry/apidocs/spectacular_ports.py

```diff
@@ -146,7 +146,16 @@ def resolve_type_hint(hint) -> Any:
         else:
             schema = resolve_type_hint(type_args[0])
         if type(None) in args:
-            schema["nullable"] = True
+            # There's an issue where if 3 types are OR'd together and one of
```

Comment:
wanna double check, what does this function look like on drf-spectacular's main branch now? do we need this shim anymore now that we're on an up to date python version?

Reply:
Hmm I think we might be able to use the original now. The [typeddict part](https://github.com/tfranzel/drf-spectacular/blob/master/drf_spectacular/plumbing.py#L1274-L1287) looks like it'll work, the only difference is it doesn't look like it excludes fields like ours. in `_resolve_typeddict`?

It seems the very recent version also now supports openapi 3.1 which drops `nullable` in favor of `null`, although I don't see the code doing this in the Union check in `resolve_type_hint` 🤔